src/Actions/authActions.js

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+import setAuthToken from "../Utils/setAuthToken";
+import jwt_decode from "jwt-decode";
+import {
+    GET_ERRORS,
+    SET_CURRENT_USER,
+} from "./types";
+// Register User
+// export const registerUser = (userData, history) => dispatch => {
+//     axios
+//         .post("/api/users/register", userData)
+//         .then(res => history.push("/login")) // re-direct to login on successful register
```

Comment:
Make sure to either delete or use if we have time to create a register page at the end